src/org/omegat/core/dictionaries/StarDict.java

```diff
@@ -147,7 +149,10 @@ public StarDict(File ifoFile) throws Exception {
                 mem.write(b);
             }
         }
-        return result;
+    }
+
+    public int entrySize() {
```

Comment:
This method name is confusing. I would expect this to mean "the size of *an* entry" but it's actually the number of entries. So I would change it to `size()` or `entryCount()` or something like that.